igor-web/src/main/groovy/com/netflix/spinnaker/igor/gitlabci/GitlabCiBuildMonitor.java

```diff
@@ -192,8 +188,8 @@ protected void commitDelta(BuildPollingDelta delta, boolean sendEvents) {
     return pipelines.stream()
         .filter(
             pipeline ->
-                (pipeline.getFinishedAt() != null)
-                    && (pipeline.getFinishedAt().getTime() > threshold))
+                (pipeline.getCreatedAt() != null)
```

Comment:
Can you help me with some terminology here.  Coming from jenkins, I create jobs and then jobs have multiple executions.  Is this talking about the creation time of the job, or an individual execution?

Reply:
Either way, seems like it's worth a comment for people expecting finishedAt that the gitlab list API doesn't return it, but we still like the list API for performance reasons.

Reply:
In Gitlab a project has pipelines which are made up of various steps, those steps are called jobs.  Its a really weird setup to be honest, a lot of the APIs are also at the job level so you make multiple calls just to get basic info on a single pipeline run.  I can add some comments.